.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*.*.*
+env:
+  GOFLAGS: -mod=readonly
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v3
+        with:
+          # Required for the changelog to work correctly.
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v4
+        with:
+          distribution: goreleaser
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.PERSONAL_GITHUB_TOKEN }}
```

Comment:
r u sure it will work?

Reply:
if you define add a/your github token as a secret to your repo, yes. 

you can see it in action on my fork: https://github.com/franklinkim/oapi-codegen/releases

Reply:
@53jk1 @franklinkim 
You can use predefined variable, passed by Github Actions, so you don't need to create new variable by yourself.
Check example [here](https://github.com/deepmap/oapi-codegen/pull/888/files#diff-82014328c5245884891cc33fbab1ee6d5fc9d2c192b8712b0a222297720172a3R28) in related issue/PR.

Reply:
Give it a try but I had to add a different one since `secrets.GITHUB_TOKEN` seemed to be missing permissions to create the release